miden-lib/src/lib.rs

```diff
@@ -46,12 +54,6 @@ impl From<MidenLib> for Library {
     }
 }
 
-impl From<MidenLib> for MastForest {
-    fn from(value: MidenLib) -> Self {
-        value.0.into()
-    }
-}
-
 impl Default for MidenLib {
     fn default() -> Self {
         let contents = Library::read_from_bytes(MIDEN_LIB_BYTES).expect("failed to read masl!");
```

Comment:
Could we update this to work similarly to how `StdLibrary::default()` works in `miden-vm`? (i.e., we use `LazyLock` there).